lib/descope/api/v1/management/audit.rb

```diff
@@ -58,6 +58,31 @@ def audit_search(
             { 'audits' => res['audits'].map { |audit| convert_audit_record(audit) } }
           end
 
+          def audit_create_event(action: nil, type: nil, data: nil, user_id: nil, actor_id: nil, tenant_id: nil)
+            # Create an audit event
+            unless %w[info warn error].include?(type)
+              raise Descope::AuthException, 'type must be either info, warn or error'
+            end
+
+            # validate data
+            raise Descope::AuthException, 'data must be provided (Hash)' unless data.is_a?(Hash)
```

Comment:
data is not required

Reply:
Yes but if passed I'm only checking that it's a hash and it's not empty

Reply:
Cool, so just add the other verifications (for action and actor)